### DISCUSSION TITLE
Will the recent changes filter through to the Home Assistant integration ?

### DISCUSSION THREAD

**Discussion by B-Hartley** [benleb/surepy#22] [General]
Will the recent changes filter through to the Home Assistant integration ?
Are you still working on that ?

Do I need to raise it within Home Assistant Github to get it updated?
It seems broken at the moment?

Thanks

**Comment by benleb** [comment]
hey... there are already at least 4 issues for this ([ha#48794](https://github.com/home-assistant/core/issues/48794), [ha#48899](https://github.com/home-assistant/core/issues/48899), [ha#49032](https://github.com/home-assistant/core/issues/49032), [ha#46449](https://github.com/home-assistant/core/issues/46449)) - but also a PR with code that uses the current version of the lib (with the fixes of course) 

you can check out the PR and test (and report problems) if you want :D it should work already (i run it here too): [ha#49223](https://github.com/home-assistant/core/pull/49223)

**Comment by B-Hartley** [comment]
Hi,

Thanks for this.
Should it just be a case of getting the surepetcare folder from components in [surepetcare-general-overhaul](https://github.com/benleb/core/tree/surepetcare-general-overhaul) and putting in my custom_components folder?

Is the config.yaml format still the same?

When I do that home assistant won't config check and if I restart it hangs with nothing in homeassistant.log

I'm sure I'm doing something stupid, any ideas ?

**Reply by benleb** [reply]
check my post here https://community.home-assistant.io/t/sureflap-connect-pet-door/41488/452?u=benleb 👍

**Comment by benleb** [comment]
https://github.com/home-assistant/core/pull/53123 first PR merged which updates to the current surepy version. Next PRs will follow in the next days :)
